sites/firehouse.com/server/templates/website-section/jobs/index.marko

```diff
@@ -0,0 +1,82 @@
+import hierarchyAliases from "@base-cms/marko-web/utils/hierarchy-aliases";
+import queryFragment from "../../../graphql/fragments/content-list";
+import GAM from "../../../../config/gam";
+
+$ const { id, alias, name, pageNode } = data;
+$ const { site } = out.global;
+
+<marko-web-website-section-page-layout id=id alias=alias name=name>
+  <@head>
+    <marko-web-gtm-website-section-context|{ context }| alias=alias>
+      <marko-web-gtm-push data=context />
+    </marko-web-gtm-website-section-context>
+    <marko-web-resolve-page|{ data: section }| node=pageNode>
+      $ const aliases = hierarchyAliases(section);
+      $ const adSlots = {
+          "gpt-ad-lb1":   GAM.getAdUnit({ name: "lb1", aliases }),
+        }
+      <marko-web-gam-slots slots=adSlots />
+    </marko-web-resolve-page>
+  </@head>
+  <@above-container>
+    <marko-web-resolve-page|{ data: section }| node=pageNode>
+      $ const aliases = hierarchyAliases(section);
+      <marko-web-gam-out-of-page-ad ...GAM.getAdUnit({ name: "wa", aliases }) />
+      <marko-web-gam-out-of-page-ad ...GAM.getAdUnit({ name: "reskin", aliases }) />
+    </marko-web-resolve-page>
+  </@above-container>
+  <@page>
+    <marko-web-gam-display-ad id="gpt-ad-lb1" modifiers=["top-of-page"] />
+    <marko-web-resolve-page|{ data: section }| node=pageNode>
+      <marko-web-page-wrapper class="mb-block">
+        <@section>
+          <div class="row mb-block">
+            <div class="col">
+              <default-theme-website-section-breadcrumbs section=section display-self=false />
+              <marko-web-website-section-name tag="h1" class="page-wrapper__title" obj=section />
+              <if(section.description)>
+                <p class="page-wrapper__deck">${section.description}</p>
+              </if>
+            </div>
+          </div>
+        </@section>
+        <@section modifiers=["bottom-border"]>
+          <div class="row mb-block">
+            <div class="col">
+              <div class="firehouse-jobs">
+                <marko-web-link href=`/${alias}/submit` class="firehouse-jobs__button">
+                  Submit A Job
+                </marko-web-link>
+              </div>
+              <p>Job posting packages are available for both standard and featured listings.</p>
+            </div>
+          </div>
+        </@section>
+        <@section class="infinite-scroll-target">
+          <div class="row">
+            <div class="col">
+              <marko-web-query|{ nodes }|
+                name="website-optioned-content"
+                params={ sectionId: id, optionName: "Pinned", limit: 250, queryFragment }
```

Comment:
This shouldn't be returning this many items at once. It should return one average screen full of content, then load the rest via the infinite scroll method (on demand). See the standard section pages or published content pages for examples of this.

If it's easier, I believe the allPublishedContent query supports limiting to only included section IDs.

Reply:
@solocommand - it was copied over from the changes Jacob made tor the MTM transit bids rfp page I did previously.  https://github.com/base-cms-websites/endeavor-business-media/pull/265/files?file-filters%5B%5D=.marko#diff-ac7602cf08ba6ff293a678a47eaf4efc

Used this for the AVP Classified's page (which is tied up in submitter review) as well. Should I update it on all 3?